### ISSUE TITLE
Unable to run/create pages with Ionic 2 RC1

### ISSUE THREAD

**Issue by suvajit** [driftyco/ionic#8854]
Hi,
  I installed Ionic 2 RC1.
  I created a blank project.

  But when I try to create new pages, I get this error 

```
LII-RLM-013:weshare suvajit$ ionic g page listings

No ionic-angular package found, do you have Ionic installed? (CLI v2.0.0)

Your system information:

Cordova CLI: 6.4.0-nightly.2016.9.7.ddd828f4 (cordova-lib@6.4.0-nightly.2016.9.7.0b574798)
Gulp version:  CLI version 3.9.1
Gulp local:  
Ionic Framework Version: 2.0.0-rc.1
Ionic CLI Version: 2.0.0
Ionic App Lib Version: 2.0.0
ios-deploy version: 1.8.6 
ios-sim version: 5.0.8 
OS: Mac OS X El Capitan
Node Version: v6.8.1
Xcode version: Xcode 7.3.1 Build version 7D1014 

```

What am I missing here ?

**Comment by ramonornela** [comment]
Run ```npm install``` it seems that the ionic-angular package not was installed

**Comment by suvajit** [comment]
Thanks.
I tried npm install but it did not help.
I guess I was having a conflict between ionic & ionic2 installations.
I took following actions : 
1. installed all ionic & cordova installation using npm uninstall
2. cleared the npm cache 
3. removed the old ionic installation in /usr/local/ionic 
4. installed fresh ionic & cordova . It got installed in /usr/local/Cellar/node/6.5.0/bin
5. created a symbolic link for ionic as /usr/local/bin/ionic
Now all is fit & fine !!
Here is the latest config for me now:
Your system information:

Cordova CLI: 6.4.0-nightly.2016.9.7.ddd828f4 (cordova-lib@6.4.0-nightly.2016.9.7.0b574798)
Gulp version:  CLI version 3.9.1
Gulp local:  
Ionic Framework Version: 2.0.0-rc.1
Ionic CLI Version: 2.1.1
Ionic App Lib Version: 2.1.1
Ionic App Scripts Version: 0.0.36
ios-deploy version: 1.8.6 
ios-sim version: 5.0.8 
OS: Mac OS X El Capitan
Node Version: v6.8.1
Xcode version: Xcode 7.3.1 Build version 7D1014
